service/internal/fixtures/policy_fixtures.yaml

```diff
@@ -615,3 +615,49 @@ namespace_key_map:
       key_id: 68fb7e3e-6c7b-434d-bf38-07701ad8655a
     - namespace_id: 87ba60e1-da12-4889-95fd-267968bf0896 # deactivated
       key_id: d433e903-e3d1-4823-9b3e-7b188e7c6060
+
+##
+# Registered Resources
+#
+##
+registered_resources:
+  metadata:
+    table_name: registered_resources
+    columns:
+      - id
+      - name
+  data:
+    res_with_values:
+      id: f3a1b2c4-5d6e-7f89-0a1b-2c3d4e5f6789
+      name: res1
```

Comment:
Totally not required, but I like the pattern you replicated below where the key is the same as the name/value so in test logs you know more quickly which fixture to look at.

Reply:
Done